src/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -212,10 +212,93 @@ public void Path_Longer_Than_MaxLongPath_Throws_Exception()
             });
         }
 
+        [Fact]
+        public void MoveDirectory_ThrowIOExceptionWhenDirectoryExists()
```

Comment:
This is testing something more specific? "when directory exists and destination is the same" .. those seem like 2 cases?

Reply:
Agreed, something along the lines of `ThrowIOExceptionWhenMovingToADirectoryThatAlreadyExists`? Incredibly wordy, but I running low on ideas on how to shorten it

Reply:
I meant, this test is testing two things at once  -- moving a directory which exists to a different directory that already exists and also moving a directory which exists to itself. Seems like those ought to be different tests, as typicaly the implementation would check for "destination exists" and "destination same as source" separately in the code. (I didn't look)

Reply:
Ha, @danmosemsft I was just in the middle of replying that I misread your comment. I should absolutely split this into two test cases with better names.